### PR TITLE
SC-8250 - Connect UI for bulk delete

### PR DESCRIPTION
**Pull request by uidp**
# Description
add bulk delete functionality to admin tables

## Links to Tickets or other pull requests
- https://ticketsystem.hpi-schul-cloud.org/browse/SC-8250
- https://github.com/hpi-schul-cloud/schulcloud-server/pull/2309

## Changes
- delete multiple users in chunks reduce server load
- show progress indicator

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Comment by uidp** [comment]
@schul-cloud-bot update screenshots

File: src/components/molecules/ProgressModal.vue [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	</base-modal>
+</template>
+<script>
+import ModalBodyInfo from "@components/molecules/ModalBodyInfo";
+export default {
+	components: {
+		ModalBodyInfo,
+	},
+	props: {
+		color: {
```
Comment (pkleybolte) [inline src/components/molecules/ProgressModal.vue:16]: unused prop

File: src/pages/administration/teachers/index.vue [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
+		<progress-modal
+			:active="isDeleting"
+			:percent="deletedPercent"
+			:title="$t('pages.administration.teachers.index.remove.progress.title')"
+			:description="
+				$t('pages.administration.teachers.index.remove.progress.description')
+			"
+			data-testid="progress-modal"
+		/>
+		<progress-modal
```
Comment (pkleybolte) [inline src/pages/administration/teachers/index.vue:13]: Second modal without translations. Please remove

File: src/serverMiddleware/routes.js [hunk 3]
```diff
@@ -16,10 +16,10 @@ const routes = [
 	`^/account/locale/edit/?$`,
 	`^/activation/${activationCode}/?$`,
 	// `^/administration/datasources?`,
-	// `^/administration/students/?$`,
+	`^/administration/students/?$`,
```
Comment (pkleybolte) [inline src/serverMiddleware/routes.js:19]: For local testing only. Needs to be removed before merging.

**Review by pkleybolte — CHANGES_REQUESTED** [review]

File: src/pages/administration/teachers/index.vue (hunk 2 above)
Comment (uidp) [reply src/pages/administration/teachers/index.vue]: done.

File: src/serverMiddleware/routes.js (hunk 3 above)
Comment (uidp) [reply src/serverMiddleware/routes.js]: done.

**Review by pkleybolte — APPROVED** [review]

**Pull request merged**